src/applications/claims-status/components/claim-document-request-pages/Automated5103Notice.jsx

```diff
@@ -1,53 +1,157 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import { connect } from 'react-redux';
 import { Link } from 'react-router-dom-v5-compat';
-import { buildDateFormatter } from '../../utils/helpers';
+import {
+  VaCheckbox,
+  VaButton,
+} from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { buildDateFormatter, setDocumentTitle } from '../../utils/helpers';
+import {
+  // START ligthouse_migration
+  submit5103 as submit5103Action,
+  submitRequest as submitRequestAction,
+  // END lighthouse_migration
+} from '../../actions';
+// START lighthouse_migration
+import { cstUseLighthouse } from '../../selectors';
+// END lighthouse_migration
+import { setUpPage } from '../../utils/page';
 
-export default function Automated5103Notice({ item }) {
-  const formattedDueDate = buildDateFormatter()(item.suspenseDate);
-  if (item.displayName !== 'Automated 5103 Notice Response') {
-    return null;
+import withRouter from '../../utils/withRouter';
+
+class Automated5103Notice extends React.Component {
```

Comment:
Why did you change this to be a class based component? We should be avoiding class based components as much as possible as we will have to convert this to a functional component in the future

Reply:
@jerekshoe I wasnt aware that we were trying to avoid class based components? I had to update also of this component to match the logic in  `AskVAPage.jsx` which is using a class so thats why I just updated it to be the same. I can change it to a functional component if this is something that we are trying to push for LMK.

Reply:
Yeah I'd swap it back so that we can avoid making more work for ourselves in the future. We can pair on it if you would like

Reply:
updated!